.github/workflows/ansible.yml

```diff
@@ -12,7 +12,12 @@ jobs:
       - name: Install all requirements
         run: |
           python3 -m pip install --upgrade -r requirements.txt
+          ansible-galaxy install -r requirements.yml
```

Comment:
What is interesting is that calling ansible-playbook also need al Ansible dependency to be ok. So this test is also an indirect test for the Ansible dependency.